src/core/controls/DropDownList.js

```diff
@@ -0,0 +1,432 @@
+                    }.bind(this));
+
+                    background.on('mouseout',function() {
+                        this.handleEvent(DropDownList.NORMAL);
+                    }.bind(this));
+
+                    background.addChild(itemText);
+                    inner.addChild(background);
+                }
+                else{
+                    var container = new PIXI.Container();
+
+                    itemText.x = 5;
+                    itemText.y = 10 + i * 40;
+
+                    container.hitArea = new PIXI.Rectangle(0, 5 + i * 40, 240, 40);
+
+                    container.interactive = true;
+                    container.click = this.selectDropDownElement.bind(this, itemText._text);
+                    container.mouseover = function() { //TODO add throttle. cant use mouseenter
+                        this.handleEvent(DropDownList.HOVER_CONTAINER, i);
+                    }.bind(this);
+
+                    container.mouseout = function() {
+                        this.handleEvent(DropDownList.NORMAL);
+                    }.bind(this);
+
+
+                    container.addChild(itemText);
+                    inner.addChild(container);
+                }
+            }.bind(this));
+
+
+            wrapper.addChild(border);
+            wrapper.addChild(inner);
+
+            this.addChild(wrapper);
+        }
+    }
+    this.updateDropDown = false;
+};
+
+
+/**
+ * Hover state: mouse pointer hovers over the VariantContainer
+ * (ignored on mobile)
+ *
+ * @property HOVER
+ * @static
+ * @final
+ * @type String
+ */
+DropDownList.HOVER_CONTAINER = 'hover_container';
+
+/**
+ * State: initial state
+ * (ignored on mobile)
+ *
+ * @property NORMAL
+ * @static
+ * @final
+ * @type String
+ */
+DropDownList.NORMAL = 'normal';
+
+/**
+ * State: clicked state
+ *
+ * @property NORMAL
+ * @static
+ * @final
+ * @type String
+ */
+DropDownList.CLICKED = 'clicked';
+
+
+/**
+ * names of possible states for a dropddown
+ *
+ * @property stateNames
+ * @static
+ * @final
+ * @type String
+ */
+DropDownList.stateNames = [
+    DropDownList.HOVER_CONTAINER,DropDownList.NORMAL,DropDownList.CLICKED
+];
+
+/**
+ * handleEvent
+ */
+DropDownList.prototype.handleEvent = function(type, option) {
+    this.handelSubscribedCallouts(type);
+
+    if(type === DropDownList.HOVER_CONTAINER){
+        if(typeof option === 'number'){
+            this.currentState = DropDownList.HOVER_CONTAINER;
+            this.hoveredElementIndex = option;
+            this.cleanChilds();
+            this.updateDropDown = true;
+            this.updateLabel = true;
+        }
+    }
+    if(type === DropDownList.NORMAL){
+        this.currentState = DropDownList.NORMAL;
+        this.hoveredElementIndex = null;
+        this.cleanChilds();
+        this.updateDropDown = true;
+        this.updateLabel = true;
+    }
+};
+
+/**
+ * override events handlers
+ */
+DropDownList.prototype.mousedown = function(event) {
+    event.stopPropagation();
```

Comment:
stop propagation on mouse/touch down will break one of our business applications where you can move the PickerList.